cfme/tests/test_ui_problems.py

```diff
@@ -12,7 +12,8 @@
 TOLERANCE = 20
 # TODO: Replace nav branch strings with navmazing object destinations when supported
 LOCATIONS = [
-    "control_explorer", "automate_explorer", "automate_customization", "my_services",
+    "control_explorer", "automate_explorer", "automate_customization",
```

Comment:
I like the style, and have seen/used it in some other PRs, but would like to leave it out of this PR to control scope.